src/main/java/seedu/mark/model/autotag/BookmarkTagger.java

```diff
@@ -0,0 +1,48 @@
+package seedu.mark.model.autotag;
```

Comment:
Why is there a need to separate `BookmarkTagger` and `SelectiveBookmarkTagger`? Can we combine them